File: java/src/main/java/com/iteso/nintendo/Flamethrower.java
package com.iteso.nintendo;
/**
 * Class that specifyes the attack type to the pokemon.
 */
public class Flamethrower implements iAttack {

    /**
     *
     * @return name of the attack.
     */
    public final String pokeAttack() {
        String attack = "Flamethrower";
        return attack;
    }

    /**
     *
     * @return the power of the attack damage.
     */
    public int power() {
        final int power = 15;
        return power;
    }
}

File: python/src/main/python/attacks/Flamethrower.py
from Attack import *


class Flamethrower(Attack):

    def __init__(self, damage=60):
        self.damage = damage

    def performAttack(self):
        return "Flamethrower"

    def getAttackDamage(self):
        return self.damage

    def setAttackDamage(self, damage):
        self.damage = damage
